README.md

```diff
@@ -127,7 +127,7 @@ To enable authenticated requests, pass an `auth` option to the Octokit construct
 
 ```js
 const clientWithAuth = new Octokit({
-  auth: 'token secret123'
+  auth: 'Bearer secret123'
```

Comment:
`Bearer` is used when authenticing as a GitHub Application, otherwise it’s token. But I think they might be interchangable, they at least are for the GraphQL API, I just tried that today. But `token` is definitely right here and more common in GitHub’s own API